models/pull.go

```diff
@@ -422,7 +422,7 @@ func (pr *PullRequest) setMerged() (err error) {
 	if err = pr.Issue.changeStatus(sess, pr.Merger, pr.Issue.Repo, true); err != nil {
 		return fmt.Errorf("Issue.changeStatus: %v", err)
 	}
-	if _, err = sess.Id(pr.ID).Cols("has_merged").Update(pr); err != nil {
+	if _, err = sess.Id(pr.ID).Cols("has_merged, merged_commit_id, merger_id, merged_unix").Update(pr); err != nil {
```

Comment:
Shouldn't it be `sess.ID` ?

Reply:
They are aliases but by GoLang naming ID is more correct

Reply:
Also, `Id(..)` is deprecated, so I think we should prefer `ID(..)` (https://godoc.org/github.com/go-xorm/xorm#Engine.Id)

Reply:
Umm ... but `Id(..)` is used in many other places.
It seems to be reasonable to fix all of them at once as an other issue.

Reply:
To not to accidently break something it's better to change them one by one when touching old code like in this case

Reply:
Okay, I'll fix it